.github/ISSUE_TEMPLATE/6_infrastructure.yml

```diff
@@ -0,0 +1,69 @@
+name: The CI infrastructure used by Flutter has a problem
+description: |
+  As a contributor, you want to file an issue about the build/test/release
+  infra, e.g. dashboards (http://flutter-dashboard.appspot.com), devicelab,
+  LUCI (https://ci.chromium.org/p/flutter) etc.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        It looks like you found an issue with our Infrastructure services.
+        Please complete the form below so that we can help to resolve your
+        issue quickly.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      options:
+      - label: I have searched the [existing issues](https://github.com/flutter/flutter/issues)
```

Comment:
Nit: https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3Ateam-infra

Reply:
Ah good catch!